### PR TITLE
Add new terms "HDAC-6 Inhibitor" and "RGFP136"

### PR DESCRIPTION
**Pull request by YojanaGadiya**
- [x] Add synonyms mentioned in CTD (http://ctdbase.org/detail.go?type=chem&acc=C556032)
- [x] Add this as a synonym N-(6-(2-amino-4-fluorophenylamino)-6-oxohexyl)-4-methylbenzamide
- [x] Add xref to MESH supplementary concept: [mesh:C556032](https://meshb.nlm.nih.gov/record/ui?ui=C556032)
- [x] Check if this is the same https://pubchem.ncbi.nlm.nih.gov/compound/rg2833#section=2D-Structure
- [x] Identify SMILES and InChI for this molecule

### CODE REVIEW

**Comment by cthoyt** [comment]
I updated the description with a checklist. These all must be done before accepting the PR

**Comment by YojanaGadiya** [comment]
@cthoyt i can use the mesh term also if you would prefer that over conso.

**Comment by cthoyt** [comment]
No this is fine, but can you find the smiles string and InChI for it too?

**Comment by YojanaGadiya** [comment]
@cthoyt  no i couldn't find any.

File: synonyms.tsv [hunk 1]
```diff
@@ -1363,3 +1363,4 @@ CONSO00352	Mini-Mental_Status_Exam	ncit:C74982	EXACT
 CONSO00353	1-(5-acetylamino-5-carboxypentyl)-3-hydroxy-5-hydroxymethyl-pyridinium	pmid:12784645	EXACT
 CONSO00353	Glyceraldehyde derived Pyridinium compound	pmid:12784645	EXACT
 CONSO00353	Glyceraldehyde-derived Pyridinium compound	pmid:12784645	EXACT
+CONSO00363	N-(6-(2-amino-4-fluorophenylamino)-6-oxohexyl)-4-methylbenzamide	ctd	EXACT
```
Comment (cthoyt) [inline synonyms.tsv]: Not a CTD identifier. That's an IUPAC name. Put in the synonyms file

File: terms.tsv [hunk 2]
```diff
@@ -361,3 +361,4 @@ CONSO00359	Sandra	TDP-43 aggregates	cellular component	pmid:30824544	TDP-43 aggr
 CONSO00360	Yojana	Clonal Expansion	pathology	pmid:27794524	Multiplication or reproduction by cell division of a population of identical cells descended from a single progenitor. In immunology, may refer to the clonal proliferation of cells responsive to a specific antigen as part of an immune response.
 CONSO00361	Charlie	CDR - Sum of Boxes	clinical measurement	pmid:25468261, pmid:23626081, pmid:27084223	Clinical Dementia Rating (CDR) Sum of Boxes
 CONSO00362	Yojana	Tau antibody, MC1	antibody	pmid:28148553, pmid:23638068, pmid:21841002	mouse monoclonal antibody specific to tau epitopes within 312 to 322 amino acid
+CONSO00363	Yojana	RGFP136	chemical	pmid:23356410, pmid:21228185	HDAC3 inhibitor
```
Comment (cthoyt) [inline terms.tsv]: Please add another term for HDAC3 inhibitor and add the appropriate relationship between these terms in relations.tsv

**Comment by cthoyt** [comment]
Notes on how to find the structure when you can't find the molecule in PubChem or another good chemistry database:

1. Google IUPAC to SMILES converter. I found https://opsin.ch.cam.ac.uk/
2. Paste the IUPAC name in here
3. Google the molecule. Go to image search, you should find the structure. Is this the same as what the IUPAC converter got?
4. If that doesn't work, read some of the papers that the Google search for the molecule came up with. Is this the same as the IUPAC converter got?
5. If that doesn't work, google "Structure to SMILES". There are a couple good ones. I've used the [PubChem one](https://pubchem.ncbi.nlm.nih.gov/edit2/index.html) before.
6. Draw the molecule in there, based on what you found in one of the papers. Don't make mistakes, nobody will ever be motivated to fix it. **DON'T MAKE MISTAKES**. Now you can get the SMILES string. Make sure you post lots of links to literature about this molecule because if someone needs to check then they will have to compare the structures in these papers to the one you've drawn

**Comment by cthoyt** [comment]
@sgebel please also see these instructions so you can share with others in the future

File: synonyms.tsv (hunk 1 above)
Comment (cthoyt) [reply synonyms.tsv]: CTD doesn't issue identifiers for molecules. They're all references in MeSH

Comment (YojanaGadiya) [reply synonyms.tsv]: @cthoyt please check if this is how you wanted the entry in synonyms file.

File: terms.tsv [hunk 3]
```diff
@@ -361,3 +361,5 @@ CONSO00359	Sandra	TDP-43 aggregates	cellular component	pmid:30824544	TDP-43 aggr
...
+CONSO00364	Yojana	HDAC-3 inhibitor	chemical	pmid:23356410, pmid:21228185	Compounds that inhibit Histone Deactelylase 3 (HDAC-3)
```
Comment (cthoyt) [inline terms.tsv]: Switch type to [chemical role](https://github.com/pharmacome/conso/blob/master/classes.tsv#L12)

File: relations.tsv [hunk 4]
```diff
@@ -77,3 +77,4 @@ CONSO	CONSO00321	Tau aggregation inhibitor	decreases	CONSO	CONSO00006	Tau aggreg
 CONSO	CONSO00354	Tau 6D	has_reference_protein	uniprot	P10636	TAU_HUMAN
 CONSO	CONSO00355	Tau 6P	has_reference_protein	uniprot	P10636	TAU_HUMAN
 MESH	D000544	Alzheimer Disease	is_a	CONSO	CONSO00234	4R tauopathy
+CONSO	CONSO00363	 RGFP136	is_a	CONSO	CONSO00364	HDAC-3 inhibitor
```
Comment (cthoyt) [inline relations.tsv]: Switch relationship to “has_role” (double check that’s the name of the relation that’s used for chemical to chemical roles

File: synonyms.tsv [hunk 5]
```diff
@@ -1363,3 +1363,4 @@ CONSO00352	Mini-Mental_Status_Exam	ncit:C74982	EXACT
 CONSO00353	1-(5-acetylamino-5-carboxypentyl)-3-hydroxy-5-hydroxymethyl-pyridinium	pmid:12784645	EXACT
 CONSO00353	Glyceraldehyde derived Pyridinium compound	pmid:12784645	EXACT
 CONSO00353	Glyceraldehyde-derived Pyridinium compound	pmid:12784645	EXACT
+CONSO00363	N-(6-(2-amino-4-fluorophenylamino)-6-oxohexyl)-4-methylbenzamide	mesh:C556032	EXACT
```
Comment (cthoyt) [inline synonyms.tsv]: Add the other lexical variants mentioned in CTD with different spacing and dashes

File: terms.tsv [hunk 6]
```diff
@@ -361,5 +361,7 @@ CONSO00359	Sandra	TDP-43 aggregates	cellular component	pmid:30824544	TDP-43 aggr
 CONSO00360	Yojana	Clonal Expansion	pathology	pmid:27794524	Multiplication or reproduction by cell division of a population of identical cells descended from a single progenitor. In immunology, may refer to the clonal proliferation of cells responsive to a specific antigen as part of an immune response.
 CONSO00361	Charlie	CDR - Sum of Boxes	clinical measurement	pmid:25468261, pmid:23626081, pmid:27084223	Clinical Dementia Rating (CDR) Sum of Boxes
 CONSO00362	Yojana	Tau antibody, MC1	antibody	pmid:28148553, pmid:23638068, pmid:21841002	mouse monoclonal antibody specific to tau epitopes within 312 to 322 amino acid
-CONSO00363	Yojana	RGFP136	chemical	pmid:23356410, pmid:21228185	HDAC3 inhibitor
-CONSO00364	Yojana	HDAC3 inhibitor	chemical role	pmid:23356410, pmid:21228185	Compounds that inhibit Histone Deactelylase 3(HDAC-3)
+CONSO00363	Yojana	DC8E8	antibody	pmid:4255369, pmid:25478017	recombinant mouse antibody specifc to MAPT
+CONSO00364	Yojana	DC11	antibody	pmid:4255369, pmid:12544837	monoclonal mouse antibody specifc to MAPT
+CONSO00366	Yojana	RGFP136	chemical	pmid:23356410, pmid:21228185	HDAC3 inhibitor
```
Comment (cthoyt) [inline terms.tsv]: This is going to fail because it's missing 355

Comment (YojanaGadiya) [reply terms.tsv]: @cthoyt Ya just updated it now.

**Comment by YojanaGadiya** [comment]
@cthoyt ready for merge.

**Comment by cthoyt** [comment]
Make a new export. There's all sorts of git garbage in here.

**Comment by cthoyt** [comment]
Please take care of this one today

**Comment by YojanaGadiya** [comment]
@cthoyt i will be in Fraunhofer tomorrow and hence will look at it tomorrow itself. Will most probably do it by this week.

**Comment by YojanaGadiya** [comment]
@cthoyt  it seems fine to me. Please have a look and merge it to the master. If any issues, let me know.

**Pull request merged**